util/chplenv/chpl_mem.py

```diff
@@ -3,23 +3,35 @@
 import sys
 
 import chpl_platform, overrides
-from utils import error, memoize
-
+from utils import error, memoize, warning
 
 @memoize
 def get(flag='host'):
-    if flag == 'host':
-        mem_val = 'cstdlib'
-    elif flag == 'target':
-        mem_val = overrides.get('CHPL_MEM')
-        if not mem_val:
-            platform_val = chpl_platform.get('target')
-            cygwin = platform_val.startswith('cygwin')
-
-            if cygwin:
-                mem_val = 'cstdlib'
-            else:
-                mem_val = 'jemalloc'
+    platform_val = chpl_platform.get('target')
```

Comment:
`'target'` -> `flag` here too?

Reply:
Yes good catch. I grepped `chpl_mem` and `chpl_jemalloc` for `.get('target')` and I think that was the last remaining place.